docs/framework/install/index.md

```diff
@@ -31,4 +31,4 @@ In general, a higher .NET Framework version will run an application that was bui
 
 ## Unsupported Windows versions
 
-- [Windows XP](on-windows-xp.md)
\ No newline at end of file
+- [Windows XP](on-windows-xp.md)
```

Comment:
Since now this is the main entrance to installation, it also needs to be changed to point to the for developers guide. I'd also remove the intro and the first sub-header, since they don't seem to add much value.